model/models.js

```diff
@@ -50,4 +50,26 @@ const fetchReviewById = (reviewId) =>
         return Promise.reject({status: 400, message: 'Invalid input!'});
     
 }
-module.exports = { fetchAllCategories, fetchAllReviews, fetchReviewById };
\ No newline at end of file
+
+const fetchCommentByReviewId = (reviewId) =>
+{
+    const selectCommentByReviewId = 
+    `SELECT * FROM comments
+    WHERE review_id = $1
+    ORDER BY created_at DESC;`;
+
+    return db.query(selectCommentByReviewId, [reviewId])
+    .then(({ rows, rowCount }) =>
+    {
+        if(rowCount === 0)
+            return Promise.reject({status: 404, message: 'review_id not found!'});
```

Comment:
review_id could be found but have no comments, we need to differentiate between row count being 0 because review id doesn't exist and that it exists but has no comments.

Reply:
I've implemented the change where it check if review_id exists but has no comments, thanks!